### ISSUE TITLE
fail to import vitual machines and host system 

### ISSUE THREAD

**Issue by afeefghannam89** [nbuchwitz/icingaweb2-module-pve#9]
Hallo Nicolai,
is this failure a bug or somthing wrong with he config.

## Expected Behavior
Successful import for VMs and hosts
## Current Behavior
the result from vm import soruce 
This Import Source failed when last checked at 2020-04-28 16:33:25: Undefined index: template

under preview tab in the director
```
#0 /usr/share/icingaweb2/modules/pve/library/Pve/Api.php(130): Icinga\Application\ApplicationBootstrap->Icinga\Application\{closure}(Integer, String, String, Integer, Array)
#1 /usr/share/icingaweb2/modules/pve/library/Pve/ProvidedHook/Director/ImportSource.php(33): Icinga\Module\Pve\Api->getVMs(Boolean)
#2 /usr/share/icingaweb2/modules/director/library/Director/Web/Table/ImportsourceHookTable.php(100): Icinga\Module\Pve\ProvidedHook\Director\ImportSource->fetchData()
#3 /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/SimpleQueryBasedTable.php(29): Icinga\Module\Director\Web\Table\ImportsourceHookTable->prepareQuery()
#4 /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/SimpleQueryBasedTable.php(15): gipfl\IcingaWeb2\Table\SimpleQueryBasedTable->getQuery()
#5 /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/QueryBasedTable.php(56): gipfl\IcingaWeb2\Table\SimpleQueryBasedTable->getPaginationAdapter()
#6 /usr/share/icingaweb2/modules/incubator/vendor/gipfl/icingaweb2/src/Table/QueryBasedTable.php(255): gipfl\IcingaWeb2\Table\QueryBasedTable->getPaginator(Object(gipfl\IcingaWeb2\Url))
#7 /usr/share/icingaweb2/modules/director/application/controllers/ImportsourceController.php(149): gipfl\IcingaWeb2\Table\QueryBasedTable->renderTo(Object(Icinga\Module\Director\Controllers\ImportsourceController))
#8 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(507): Icinga\Module\Director\Controllers\ImportsourceController->previewAction()
#9 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch(String)
#10 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#11 /usr/share/php/Icinga/Application/Web.php(300): Zend_Controller_Front->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#12 /usr/share/php/Icinga/Application/webrouter.php(99): Icinga\Application\Web->dispatch()
#13 /usr/share/icingaweb2/public/index.php(4): require_once(String)
#14 {main}
```

the result from host import soruce
This Import Source failed when last checked at 2020-04-28 16:32:52: Undefined index: maxcpu

![image](https://user-images.githubusercontent.com/39904920/80502769-53e44500-8971-11ea-9c9d-af277b093749.png)

## Context
How has this issue affected you? I make a monitoring project for my final exam and the pve-modul is an important part from it.  

## Your Environment
director	       1.7.2
incubator	       0.5.0
ipl	               v0.5.0
monitoring	2.7.3
pve	                0.1.0
reactbundle	0.7.0

**Comment by afeefghannam89** [comment]
Hallo again,

i have cluster from two Proxmox nodes. One of them was down. After i started him the sync for my target node 192.168.100.50 worked. After that when shut down him again the sync stay ok. Why should all nodes in a cluster be up with the first sync from host system type? Also the failure doesn't refer to this but to a missing index.

The sync for VMs worked okay after i changed the "key column name" to vm_id and the both nodes were up. But when the not related PVE host goes down, the import from the targeted PVE host crashed again.
I use Proxmox VE 6.1-3 version.
![image](https://user-images.githubusercontent.com/39904920/80524680-4d18fa80-8990-11ea-9570-9b37c54acb00.png)
The vm 103 is on the down node, which is not related to this import.

**Comment by nbuchwitz** [comment]
Hi Afeef,

looks like some fields are missing in the PVE API when the cluster is in a failed state. Could you please share the output of the following command for each of the 3 states (both only, A only, B only):

`pvesh  get /cluster/resources --output-format json`

Nicolai

**Comment by afeefghannam89** [comment]
Hi Nicolai,
here you are:)

Output with both
[{"id":"/pool/testpool","pool":"testpool","type":"pool"},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/101","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7-node1","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":101},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/103","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Centos-node2-without-migration","netin":0,"netout":0,"node":"proxmox-node2","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":103},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"lxc/102","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Container1","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":0,"type":"lxc","uptime":0,"vmid":102},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/110","maxcpu":1,"maxdisk":0,"maxmem":536870912,"mem":0,"name":"VM 110","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":110},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/100","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":100},{"cpu":0.0530616179999962,"disk":3388116992,"id":"node/proxmox","level":"","maxcpu":2,"maxdisk":7595335680,"maxmem":1996390400,"mem":1041403904,"node":"proxmox","status":"online","type":"node","uptime":5256},{"cpu":0.0326168659217408,"disk":3106308096,"id":"node/proxmox-node2","level":"","maxcpu":1,"maxdisk":7595335680,"maxmem":1996693504,"mem":960917504,"node":"proxmox-node2","status":"online","type":"node","uptime":5258},{"disk":3388116992,"id":"storage/proxmox/local","maxdisk":7595335680,"node":"proxmox","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":3106308096,"id":"storage/proxmox-node2/local","maxdisk":7595335680,"node":"proxmox-node2","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":4579850295,"id":"storage/proxmox/local-lvm","maxdisk":13954449408,"node":"proxmox","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":1654997699,"id":"storage/proxmox-node2/local-lvm","maxdisk":13954449408,"node":"proxmox-node2","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":0,"id":"storage/proxmox/lvm-thin-neu","maxdisk":106296246272,"node":"proxmox","shared":0,"status":"available","storage":"lvm-thin-neu","type":"storage"},{"disk":31372,"id":"storage/proxmox/zfsimage","maxdisk":101171828780,"node":"proxmox","shared":0,"status":"available","storage":"zfsimage","type":"storage"},{"disk":2051670016,"id":"storage/proxmox/zfsiso-backup","maxdisk":103223394304,"node":"proxmox","shared":1,"status":"available","storage":"zfsiso-backup","type":"storage"}]

Output with only A
[{"id":"/pool/testpool","pool":"testpool","type":"pool"},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/110","maxcpu":1,"maxdisk":0,"maxmem":536870912,"mem":0,"name":"VM 110","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":110},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"lxc/102","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Container1","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":0,"type":"lxc","uptime":0,"vmid":102},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/100","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":100},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/101","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7-node1","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":101},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/103","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Centos-node2-without-migration","netin":0,"netout":0,"node":"proxmox-node2","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":103},{"id":"node/proxmox-node2","level":"","maxcpu":1,"maxdisk":7595335680,"maxmem":1996693504,"node":"proxmox-node2","status":"offline","type":"node"},{"cpu":0.0185599292442048,"disk":3388174336,"id":"node/proxmox","level":"","maxcpu":2,"maxdisk":7595335680,"maxmem":1996390400,"mem":1040728064,"node":"proxmox","status":"online","type":"node","uptime":5616},{"disk":1654997699,"id":"storage/proxmox-node2/local-lvm","maxdisk":13954449408,"node":"proxmox-node2","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":4579850295,"id":"storage/proxmox/local-lvm","maxdisk":13954449408,"node":"proxmox","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":0,"id":"storage/proxmox/lvm-thin-neu","maxdisk":106296246272,"node":"proxmox","shared":0,"status":"available","storage":"lvm-thin-neu","type":"storage"},{"disk":31372,"id":"storage/proxmox/zfsimage","maxdisk":101171828780,"node":"proxmox","shared":0,"status":"available","storage":"zfsimage","type":"storage"},{"disk":3106377728,"id":"storage/proxmox-node2/local","maxdisk":7595335680,"node":"proxmox-node2","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":3388174336,"id":"storage/proxmox/local","maxdisk":7595335680,"node":"proxmox","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":2051670016,"id":"storage/proxmox/zfsiso-backup","maxdisk":103223394304,"node":"proxmox","shared":1,"status":"available","storage":"zfsiso-backup","type":"storage"}]

Output with only B
[{"id":"/pool/testpool","pool":"testpool","type":"pool"},{"id":"lxc/102","node":"proxmox","pool":"testpool","status":"unknown","type":"lxc","vmid":102},{"id":"qemu/100","node":"proxmox","pool":"testpool","status":"unknown","type":"qemu","vmid":100},{"id":"qemu/110","node":"proxmox","status":"unknown","type":"qemu","vmid":110},{"id":"qemu/101","node":"proxmox","status":"unknown","type":"qemu","vmid":101},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/103","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Centos-node2-without-migration","netin":0,"netout":0,"node":"proxmox-node2","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":103},{"cpu":0,"disk":3102556160,"id":"node/proxmox-node2","level":"","maxcpu":1,"maxdisk":7595335680,"maxmem":1996709888,"mem":1102819328,"node":"proxmox-node2","status":"online","type":"node","uptime":21},{"id":"node/proxmox","node":"proxmox","status":"offline","type":"node"},{"id":"storage/proxmox/zfsiso-backup","node":"proxmox","shared":1,"status":"unknown","storage":"zfsiso-backup","type":"storage"},{"id":"storage/proxmox/lvm-thin-neu","node":"proxmox","shared":0,"status":"unknown","storage":"lvm-thin-neu","type":"storage"},{"disk":1654997699,"id":"storage/proxmox-node2/local-lvm","maxdisk":13954449408,"node":"proxmox-node2","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"id":"storage/proxmox/local-lvm","node":"proxmox","shared":0,"status":"unknown","storage":"local-lvm","type":"storage"},{"id":"storage/proxmox/zfsimage","node":"proxmox","shared":0,"status":"unknown","storage":"zfsimage","type":"storage"},{"disk":3102556160,"id":"storage/proxmox-node2/local","maxdisk":7595335680,"node":"proxmox-node2","shared":0,"status":"available","storage":"local","type":"storage"},{"id":"storage/proxmox/local","node":"proxmox","shared":0,"status":"unknown","storage":"local","type":"sto

**Comment by afeefghannam89** [comment]
Sorry the output from B is not complete here again
[{"id":"/pool/testpool","pool":"testpool","type":"pool"},{"id":"lxc/102","node":"proxmox","pool":"testpool","status":"unknown","type":"lxc","vmid":102},{"id":"qemu/100","node":"proxmox","pool":"testpool","status":"unknown","type":"qemu","vmid":100},{"id":"qemu/110","node":"proxmox","status":"unknown","type":"qemu","vmid":110},{"id":"qemu/101","node":"proxmox","status":"unknown","type":"qemu","vmid":101},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/103","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Centos-node2-without-migration","netin":0,"netout":0,"node":"proxmox-node2","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":103},{"cpu":0,"disk":3102556160,"id":"node/proxmox-node2","level":"","maxcpu":1,"maxdisk":7595335680,"maxmem":1996709888,"mem":1102819328,"node":"proxmox-node2","status":"online","type":"node","uptime":21},{"id":"node/proxmox","node":"proxmox","status":"offline","type":"node"},{"id":"storage/proxmox/zfsiso-backup","node":"proxmox","shared":1,"status":"unknown","storage":"zfsiso-backup","type":"storage"},{"id":"storage/proxmox/lvm-thin-neu","node":"proxmox","shared":0,"status":"unknown","storage":"lvm-thin-neu","type":"storage"},{"disk":1654997699,"id":"storage/proxmox-node2/local-lvm","maxdisk":13954449408,"node":"proxmox-node2","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"id":"storage/proxmox/local-lvm","node":"proxmox","shared":0,"status":"unknown","storage":"local-lvm","type":"storage"},{"id":"storage/proxmox/zfsimage","node":"proxmox","shared":0,"status":"unknown","storage":"zfsimage","type":"storage"},{"disk":3102556160,"id":"storage/proxmox-node2/local","maxdisk":7595335680,"node":"proxmox-node2","shared":0,"status":"available","storage":"local","type":"storage"},{"id":"storage/proxmox/local","node":"proxmox","shared":0,"status":"unknown","storage":"local","type":"storage"}]

**Comment by nbuchwitz** [comment]
Could you please enclose the output with code tags for better readability?

**Comment by afeefghannam89** [comment]
output with both
```json
[{"id":"/pool/testpool","pool":"testpool","type":"pool"},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/101","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7-node1","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":101},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/103","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Centos-node2-without-migration","netin":0,"netout":0,"node":"proxmox-node2","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":103},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"lxc/102","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Container1","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":0,"type":"lxc","uptime":0,"vmid":102},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/110","maxcpu":1,"maxdisk":0,"maxmem":536870912,"mem":0,"name":"VM 110","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":110},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/100","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":100},{"cpu":0.0530616179999962,"disk":3388116992,"id":"node/proxmox","level":"","maxcpu":2,"maxdisk":7595335680,"maxmem":1996390400,"mem":1041403904,"node":"proxmox","status":"online","type":"node","uptime":5256},{"cpu":0.0326168659217408,"disk":3106308096,"id":"node/proxmox-node2","level":"","maxcpu":1,"maxdisk":7595335680,"maxmem":1996693504,"mem":960917504,"node":"proxmox-node2","status":"online","type":"node","uptime":5258},{"disk":3388116992,"id":"storage/proxmox/local","maxdisk":7595335680,"node":"proxmox","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":3106308096,"id":"storage/proxmox-node2/local","maxdisk":7595335680,"node":"proxmox-node2","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":4579850295,"id":"storage/proxmox/local-lvm","maxdisk":13954449408,"node":"proxmox","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":1654997699,"id":"storage/proxmox-node2/local-lvm","maxdisk":13954449408,"node":"proxmox-node2","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":0,"id":"storage/proxmox/lvm-thin-neu","maxdisk":106296246272,"node":"proxmox","shared":0,"status":"available","storage":"lvm-thin-neu","type":"storage"},{"disk":31372,"id":"storage/proxmox/zfsimage","maxdisk":101171828780,"node":"proxmox","shared":0,"status":"available","storage":"zfsimage","type":"storage"},{"disk":2051670016,"id":"storage/proxmox/zfsiso-backup","maxdisk":103223394304,"node":"proxmox","shared":1,"status":"available","storage":"zfsiso-backup","type":"storage"}]
```
output with A
```json
[{"id":"/pool/testpool","pool":"testpool","type":"pool"},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/110","maxcpu":1,"maxdisk":0,"maxmem":536870912,"mem":0,"name":"VM 110","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":110},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"lxc/102","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Container1","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":0,"type":"lxc","uptime":0,"vmid":102},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/100","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7","netin":0,"netout":0,"node":"proxmox","pool":"testpool","status":"stopped","template":1,"type":"qemu","uptime":0,"vmid":100},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/101","maxcpu":1,"maxdisk":10737418240,"maxmem":536870912,"mem":0,"name":"Centos7-node1","netin":0,"netout":0,"node":"proxmox","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":101},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/103","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Centos-node2-without-migration","netin":0,"netout":0,"node":"proxmox-node2","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":103},{"id":"node/proxmox-node2","level":"","maxcpu":1,"maxdisk":7595335680,"maxmem":1996693504,"node":"proxmox-node2","status":"offline","type":"node"},{"cpu":0.0185599292442048,"disk":3388174336,"id":"node/proxmox","level":"","maxcpu":2,"maxdisk":7595335680,"maxmem":1996390400,"mem":1040728064,"node":"proxmox","status":"online","type":"node","uptime":5616},{"disk":1654997699,"id":"storage/proxmox-node2/local-lvm","maxdisk":13954449408,"node":"proxmox-node2","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":4579850295,"id":"storage/proxmox/local-lvm","maxdisk":13954449408,"node":"proxmox","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"disk":0,"id":"storage/proxmox/lvm-thin-neu","maxdisk":106296246272,"node":"proxmox","shared":0,"status":"available","storage":"lvm-thin-neu","type":"storage"},{"disk":31372,"id":"storage/proxmox/zfsimage","maxdisk":101171828780,"node":"proxmox","shared":0,"status":"available","storage":"zfsimage","type":"storage"},{"disk":3106377728,"id":"storage/proxmox-node2/local","maxdisk":7595335680,"node":"proxmox-node2","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":3388174336,"id":"storage/proxmox/local","maxdisk":7595335680,"node":"proxmox","shared":0,"status":"available","storage":"local","type":"storage"},{"disk":2051670016,"id":"storage/proxmox/zfsiso-backup","maxdisk":103223394304,"node":"proxmox","shared":1,"status":"available","storage":"zfsiso-backup","type":"storage"}]
```
output with B
```json
[{"id":"/pool/testpool","pool":"testpool","type":"pool"},{"id":"lxc/102","node":"proxmox","pool":"testpool","status":"unknown","type":"lxc","vmid":102},{"id":"qemu/100","node":"proxmox","pool":"testpool","status":"unknown","type":"qemu","vmid":100},{"id":"qemu/110","node":"proxmox","status":"unknown","type":"qemu","vmid":110},{"id":"qemu/101","node":"proxmox","status":"unknown","type":"qemu","vmid":101},{"cpu":0,"disk":0,"diskread":0,"diskwrite":0,"id":"qemu/103","maxcpu":1,"maxdisk":5368709120,"maxmem":536870912,"mem":0,"name":"Centos-node2-without-migration","netin":0,"netout":0,"node":"proxmox-node2","status":"stopped","template":0,"type":"qemu","uptime":0,"vmid":103},{"cpu":0,"disk":3102556160,"id":"node/proxmox-node2","level":"","maxcpu":1,"maxdisk":7595335680,"maxmem":1996709888,"mem":1102819328,"node":"proxmox-node2","status":"online","type":"node","uptime":21},{"id":"node/proxmox","node":"proxmox","status":"offline","type":"node"},{"id":"storage/proxmox/zfsiso-backup","node":"proxmox","shared":1,"status":"unknown","storage":"zfsiso-backup","type":"storage"},{"id":"storage/proxmox/lvm-thin-neu","node":"proxmox","shared":0,"status":"unknown","storage":"lvm-thin-neu","type":"storage"},{"disk":1654997699,"id":"storage/proxmox-node2/local-lvm","maxdisk":13954449408,"node":"proxmox-node2","shared":0,"status":"available","storage":"local-lvm","type":"storage"},{"id":"storage/proxmox/local-lvm","node":"proxmox","shared":0,"status":"unknown","storage":"local-lvm","type":"storage"},{"id":"storage/proxmox/zfsimage","node":"proxmox","shared":0,"status":"unknown","storage":"zfsimage","type":"storage"},{"disk":3102556160,"id":"storage/proxmox-node2/local","maxdisk":7595335680,"node":"proxmox-node2","shared":0,"status":"available","storage":"local","type":"storage"},{"id":"storage/proxmox/local","node":"proxmox","shared":0,"status":"unknown","storage":"local","type":"storage"}]
```

**Comment by rabner** [comment]
Is there any update on the issue? It seems I have the same error.
PVE 6.3-3
icinga2 2.11.2-1ubuntu3 arm64

**Comment by nbuchwitz** [comment]
Probably fixed with #19
